### PR TITLE
feat: Improve discovery of keyboard shortcuts

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Details

New keyboard users may not be familiar with the metaphor of using F6 to move between panes. To make this a bit easier to discover, we're including this information and a link to the full keyboard shortcuts in the Live Inspect pane. The link goes through a fwlink and resolves to https://accessibilityinsights.io/docs/windows/reference/keyboard/.

##### Motivation

Improve the UX for new keyboard users.

##### Context

Like many other places in the code, this uses string concatenation and is not really localizable. I tried a version that used a RegEx, but the Hyperlink styling was wrong. The same problem exists in all 3 text blocks in this control. so I'm deferring solving that problem for a later time if/when we take on proper localization.

We discussed putting this into the Welcome screen, but decided against it for the following reasons:
1. That dialog is already at its limit at 200% zoom
2. Should the user ever uncheck the "Show this dialog at startup" checkbox, we don't provide an easy way to get it back

#### Screenshots

Before | After
--- | ---
image | image

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/main/docs/Scenarios.md) completed?
- [n/a] Does this address an existing issue? 
- [x] Includes UI changes?
  - [x] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [x] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights/Properties/Resources.resx [hunk 1]
```diff
@@ -586,4 +586,10 @@ Confidence: {1}</value>
     <value>{0} 1 of 2</value>
     <comment>{0} is the text displayed in the tooltip</comment>
   </data>
+  <data name="LiveModeControl_Keyboard" xml:space="preserve">
+    <value>Use Tab or Shift+Tab to move between fields in the same pane. Use F6 or Shift+F6 to move between panes</value>
```
Comment (codeofdusk) [inline src/AccessibilityInsights/Properties/Resources.resx:590]:
```suggestion
    <value>Use Tab or Shift+Tab to move between fields in the currently active pane. Use F6 or Shift+F6 to move between panes</value>
```

**Review by codeofdusk — CHANGES_REQUESTED** [review]
I like the approach here! However, I'm a bit concerned that this isn't very discoverable – focus lands on the "to inspect a UI Automation element..." text, then I have to tab four times to reach the new hint. Could the tab/focus order be changed so that this new text is immediately focused on app launch to ease discoverability?

**Comment by DaveTryon** [comment]
> Could the tab/focus order be changed so that this new text is immediately focused on app launch to ease discoverability?

I think this would be a tough sell with design, since the navigation isn't really an option until something is either inspected or tested. I would think that a brand new user would tend to explore a bit on the first run, so requiring those first tabs doesn't seem terribly unreasonable. 

In an ideal world, I'd love to have a real tutorial experience, but that's probably unrealistic given the other things that would be competing for engineering and design budget.

**Comment by codeofdusk** [comment]
> I would think that a brand new user would tend to explore a bit on the first run, so requiring those first tabs doesn't seem terribly unreasonable.

Consider:

1. A new user launches the app and navigates out of the welcome screen.
2. The "to begin inspecting..." text is presented.
3. The user navigates away (as instructed) and presses <kbd>Shift</kbd>+<kbd>F9</kbd>.
4. The user is dropped into the UIA tree. Tabbing around is, of course, locked to the UIA tree pane. At no time can the keyboard shortcuts hint be discovered.

**Review by codeofdusk — APPROVED** [review]
Discussed offline, concerns to be addressed in a follow-up.

**Pull request merged**